src/CqlAntlrListener.ts

```diff
@@ -102,6 +108,19 @@ export default class CqlAntlrListener implements cqlListener {
       new CqlExpressionDefinitionCreator(ctx).buildDao();
 
     if (cqlCode) {
+      if (ctx.start.inputStream) {
+        const hiddenTokens = this.bufferedTokenStream.getHiddenTokensToLeft(ctx.start.tokenIndex, cqlLexer.HIDDEN)
+        let comment = "";
+        hiddenTokens.forEach((token) => {
+          if (token.text){
+            comment += token.text;
+          }
+        })
+        comment = comment.trim();
+        if (comment){
+          cqlCode.comment = comment;
```

Comment:
I think we should remove the comment characters like // and /**/, what do you think? rest looks good

Reply:
merging it to unblock other tickets. will work on this if required.